bin/sidekiq

```diff
@@ -31,7 +31,12 @@ begin
   cli.run
 rescue => e
   raise e if $DEBUG
-  STDERR.puts e.message
-  STDERR.puts e.backtrace.join("\n")
+  if Sidekiq.error_handlers.length === 0
```

Comment:
Why ===? I think you are doing too much JavaScript? 😂

Reply:
ah sorry this is my bad. I'll fix.